reciprocalspaceship/utils/binning.py

```diff
@@ -58,7 +58,10 @@ def assign_with_binedges(data, bin_edges, right_inclusive=True):
 
 
 def bin_by_percentile(
```

Comment:
If anyone has used this function in their code, this will change the default behavior. I'd move that if we're going to change the default behavior we may as well just get rid of the labels entirely at the `utils` level and migrate the format string stuff to the top level API. This has the side effect of letting the formatting be user-facing via the intended API.

Reply:
yep. I think it's fine to change the API in `utils` because it's not intended to be user-facing. I agree with you that the formatting should be handled in `DataSet.assign_resolution_bins()` instead of the `utils` function